app/assets/javascripts/admin/addon/templates/dashboard_general.hbs

```diff
@@ -12,12 +12,19 @@
               {{i18n "admin.dashboard.community_health"}}
             </a>
           </h2>
-          <PeriodChooser
-            @period={{this.period}}
-            @action={{action "changePeriod"}}
-            @content={{this.availablePeriods}}
-            @fullDay={{false}}
-          />
+          <span>
+            <PeriodChooser
+              @period={{this.period}}
+              @action={{action "changePeriod"}}
+              @content={{this.availablePeriods}}
+              @fullDay={{false}}
+            />
+            <DButton
+              @icon="cog"
+              class="custom-date-range-button"
+              @action={{action "openCustomDateRangeModal"}}
```

Comment:
The action helper isn't required when using the `@action` decorator, so we can do this instead:
```suggestion
              @action={{this.openCustomDateRangeModal}}
```